adafruit_ssd1327.py

```diff
@@ -51,11 +51,19 @@
     b"\xAF\x00"  # DISPLAY_ON
 )
 
+
 # pylint: disable=too-few-public-methods
 class SSD1327(displayio.Display):
-    """SSD1327 driver"""
+    """SSD1327 driver
+
+    :param ~displayio.I2CDisplay bus: The data bus the display is on
+    :param int height: (keyword-only) The height of the screen
+    :param int width: (keyword-only) The width of the screen
+    :param int rotation: (keyword-only) The rotation.orientation of the
```

Comment:
Looks like there's a period between rotation and orientation instead of a `/`, that's my bad.